src/status_im2/contexts/quo_preview/navigation/top_nav.cljs

```diff
@@ -2,48 +2,89 @@
   (:require [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [reagent.core :as reagent]
-            [status-im2.common.home.view :as home.view]
-            [status-im2.contexts.quo-preview.preview :as preview]))
+            [quo2.core :as quo]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [status-im2.common.resources :as resources]
+            [quo2.theme :as quo.theme]))
 
 (def descriptor
-  [{:label   "Type"
-    :key     :type
+  [{:label   "Notification"
+    :key     :notification
     :type    :select
-    :options [{:key   :default
-               :value "Default"}
-              {:key   :blur-bg
-               :value "Blurred BG"}
-              {:key   :black
-               :value "Black"}]}
-   {:label "New Notifications?"
-    :key   :new-notifications?
+    :options [{:key   :mention
+               :value "Mention"}
+              {:key   :notification
+               :value "Notification"}
+              {:key   :seen
+               :value "Seen"}
+              {:key   :false
+               :value "False"}]}
+   {:label "Blur?"
+    :key   :blur?
     :type  :boolean}
-   {:label   "Notification Indicator"
-    :key     :notification-indicator
+   {:label "Jump To?"
+    :key   :jump-to?
+    :type  :boolean}
+   {:label "Notification Count"
+    :key   :notification-count
+    :type  :number}
+   {:label   "Customization color:"
+    :key     :customization-color
     :type    :select
-    :options [{:key   :unread-dot
-               :value :unread-dot}
-              {:key   :counter
-               :value :counter}]}
-   {:label "Counter Label"
-    :key   :counter-label
-    :type  :text}])
+    :options (map (fn [color]
```

Comment:
Probably `preview/customization-color-option` can be used [from here](https://github.com/status-im/status-mobile/pull/17095)